src/components/transactions/page-top.tsx

```diff
@@ -4,14 +4,25 @@ import { useOrigin } from '@common/hooks/use-origin';
 import { useTransactionPageTitle } from '@common/hooks/transaction/use-transaction-page-title';
 import { Stack } from '@stacks/ui';
 import { Caption, Title } from '@components/typography';
+import { useCurrentNetwork } from '@common/hooks/use-current-network';
 
 export const TransactionPageTop = memo(() => {
   const transactionRequest = useTransactionRequest();
   const origin = useOrigin();
   const pageTitle = useTransactionPageTitle();
+  const network = useCurrentNetwork();
   if (!transactionRequest) return null;
 
   const appName = transactionRequest?.appDetails?.name;
+
+  const testnetAddition = network.isTestnet ? (
+    <>
+      {' '}
+      on <br />
+      {network.url}
```

Comment:
no need to display `https://`
```suggestion
      {network.url.split('//')[1]}
```

Reply:
```ts
function cleanUrl(url: string){
  return url.split('//')[1]
}
// or
function cleanUrl(url: string){
  return new URL(url).origin // i think, this is from memory lol
}
```

Reply:
`hostname`*